web_responsive/static/src/less/form_view.less

```diff
@@ -1,14 +1,48 @@
 /* Copyright 2016 Ponto Suprimentos Ltda.
+   Copyright 2018 Alexandre Díaz
  * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
 
+@sheet-margin: @sheet-padding;
+@chatter-side-width: 390px;
+
+// Sided Chatter
+.o_chatter_position_sided {
```

Comment:
My suggestion is to use flexbox here. You have a playground in https://demo.agektmr.com/flexbox/. If I'm not wrong, this should do the trick (or almost):

```less
@media (min-width: @screen-md) {
.o_chatter_position_sided {
  .o_form_view {
    display: flex;
    flex-flow: row;
    height: 100%;
    align-items: stretch;

    .o_form_sheet_bg {
      border-right: 1px solid @table-border-color;
      overflow: auto;
      flex: 3 0 auto;
    }

    .o_chatter {
      overflow: auto;
      flex: 1 1 @chatter-side-width;

      .o_chatter_topbar {
        height: auto;
        flex-wrap: wrap;
      }
    }
  }
}
}
```

The benefits:

- No JS needed. Thus, more fast.
- Adaptable to any size.
- Autofills if no chatter available.
- Cleaner, IMHO.